app/src/main/res/layout/rowlayout.xml

```diff
@@ -27,15 +28,15 @@
         android:layout_marginTop="5dp"
         android:layout_weight="1"
         android:textSize="18dp"/>
+
     <TextView
-        android:id ="@+id/rcpIngredientsTitle"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:id="@+id/rcpIngredientsTitle"
+        android:layout_width="194dp"
+        android:layout_height="43dp"
```

Comment:
Avoid setting specific sizes.  That makes your app dependent on a specific resolution.   Also reduces accessibility.

Better to use relative (as it was before).

Reply:
Acknowledge.